packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -524,27 +637,42 @@ class AndroidCameraCameraX extends CameraPlatform {
         (offset / exposureOffsetStepSize).round();
 
     try {
```

Comment:
Small fix here: Before I was returning a value based on my own computations, but we should actually just use the value that directly comes from CameraX to be most correct.